lib/membrane_h264_ffmpeg/encoder.ex

```diff
@@ -68,14 +68,20 @@ defmodule Membrane.H264.FFmpeg.Encoder do
                 to decode video encoded with this element.
                 """,
                 type: :atom,
-                spec: H264.profile_t(),
-                default: :high
+                spec: H264.profile_t() | :auto,
+                default: :auto
               ],
               use_shm?: [
                 type: :boolean,
                 desciption:
                   "If true, native encoder will use shared memory (via `t:Shmex.t/0`) for storing frames",
                 default: false
+              ],
+              max_b_frames: [
+                type: :int,
+                description:
+                  "Maximum number of B-frames between non-B-frames. Set to 0 to encode video without b-frames",
+                default: -1
```

Comment:
Using magic value is more C-style but should rather be avoided in Elixir. If not set, this should be `nil` and should be mapped to a value more convenient for native code inside the element